src/styles/index.scss

```diff
@@ -36,7 +50,7 @@
 // @import '~bootstrap/scss/badge';
 @import '~bootstrap/scss/alert';
 // @import '~bootstrap/scss/progress';
-// @import '~bootstrap/scss/list-group';
+@import '~bootstrap/scss/list-group';
```

Comment:
Just as a thought, for something like this to be useful it probably shouldn't be dependent on the entire BS style library. 

Two things come to mind:

1) style it without bootstrap?
2) an optional bootstrap mode?

Reply:
@PenguinOfWar Good suggestion - I'm happy to style without bootstrap or / copy the bootstrap css keeping the names and properties of the classes so IF someone were to use bootstrap they could modify the classes in their own sass. Removes the dependancy on it and makes it work the same way. What do you think?

Reply:
I would give it generic classes that can be styled with custom CSS, and maybe the option to enable default bootstrap classes. Adding overrides is also pretty doable but may be too much if you're already adding your own, sensible, generic classes.